vendor/bat-native-ads/src/bat/ads/internal/ad_targeting/resources/frequency_capping/anti_targeting_resource.cc

```diff
@@ -0,0 +1,120 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/ad_targeting/resources/frequency_capping/anti_targeting_resource.h"
+
+#include <utility>
+#include <vector>
+
+#include "base/json/json_reader.h"
+#include "bat/ads/internal/ads_client_helper.h"
+#include "bat/ads/internal/logging.h"
+#include "bat/ads/result.h"
+#include "brave/components/l10n/common/locale_util.h"
+
+namespace ads {
+namespace resource {
+
+namespace {
+const int kCurrentVersion = 1;
+const char kResourceId[] = "mkdhnfmjhklfnamlheoliekgeohamoig";
+}  // namespace
+
+AntiTargeting::AntiTargeting() = default;
+
+AntiTargeting::~AntiTargeting() = default;
+
+bool AntiTargeting::IsInitialized() const {
+  return is_initialized_;
+}
+
+void AntiTargeting::Load() {
+  AdsClientHelper::Get()->LoadUserModelForId(
+      kResourceId, [=](const Result result, const std::string& json) {
+        if (result != SUCCESS) {
+          BLOG(1, "Failed to load resource " << kResourceId);
+          is_initialized_ = false;
+          return;
+        }
+
+        BLOG(1, "Successfully loaded resource " << kResourceId);
```

Comment:
nit: Successfully loaded anti-targeting resource ... (same for other logs)

Reply:
file name shows in log, so keeping as is for now